local-volume/provisioner/pkg/controller/controller.go

```diff
@@ -49,6 +50,11 @@ func StartLocalController(client *kubernetes.Clientset, ptable deleter.ProcTable
 	broadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: v1core.New(client.CoreV1().RESTClient()).Events("")})
 	recorder := broadcaster.NewRecorder(scheme.Scheme, v1.EventSource{Component: provisionerName})
 
+	// We choose a random resync period between MinResyncPeriod and 2 *
+	// MinResyncPeriod, so that local provisioners deployed on multiple nodes
+	// at same time don't list the apiserver simultaneously.
+	resyncPeriod := time.Duration(float64(config.MinResyncPeriod.Nanoseconds()) * (1 + rand.Float64()))
```

Comment:
The final value should be multiplied by time.Nanosecond.  Also, do we really need Nanosecond granularlity? I think probably Seconds should be sufficient.

Reply:
Fixed in https://github.com/kubernetes-incubator/external-storage/pull/800/commits/fec4ef952a9006121e3b5ac61b3149cb2eb46b38. Also initialized default source in main.go.